project-base/storefront/public/locales/en/common.json

```diff
@@ -89,9 +89,11 @@
     "Customer information": "Customer information",
     "Customized E-shop by": "Customized E-shop by",
     "Czechia": "Czechia",
+    "date of order": "",
```

Comment:
You are missing English strings for essentially all added keys.

Reply:
Fixed